invisible_cities/evm/nh5.py

```diff
@@ -43,6 +43,12 @@ class MCExtentInfo(tb.IsDescription):
     last_particle = tb.UInt64Col(pos=2)
 
 
+class MCEventMap(tb.IsDescription):
+    """Map between event index and original event."""
```

Comment:
I found this description and the row names below a bit misleading. After reading the code and discussing it with @andLaing, I understand that `evt_number` is always going to be the event number read from the NEXUS file, and that `nexus_evt` is actually a subevent-like index unrelated to NEXUS. I'd change the docstring and row names to reflect that.